components/prism-wolfram.js

```diff
@@ -0,0 +1,24 @@
+Prism.languages.wolfram = {
+	'comment': {
+		pattern: /(?:\(\*[\w\'\s\r\n\*]*\*\))/
+	},
```

Comment:
1. You can inline the regex.
2. Unnecessary non-capturing group.
2. `[\w\'\s\r\n\*]*`: typically, languages allow pretty much all characters inside comments. Wouldn't `\(\*[\s\S]*?\*\)` work better?

Also, asking out of curiosity, does the language support nested comments? I.e. `(* comment (* nested *) *)`.

```suggestion
	'comment': /\(\*[\s\S]*?\*\)/,
```

Reply:
@RunDevelopment Yes it supports nested comments, anything within the enclosing (* *) is considered a comment

Reply:
> Yes it supports nested comments

Ah, ok. In that case, it's the same [comment syntax as Applescript](https://github.com/PrismJS/prism/blob/ad9878ad39cd7afaab9e3a67f99fef3d8167b28c/components/prism-applescript.js#L4). You can probably just copy-paste the regex.

Reply:
Oh nice, I'll update to that.